.github/workflows/dax.yml

```diff
@@ -12,6 +22,8 @@ permissions:
 
 env:
   UMF_TESTS_DEVDAX_NAMESPACE : "0.0"
+  UMF_TESTS_FSDAX_NAMESPACE : "1.0"
```

Comment:
you could drop this `UMF_TESTS_` prefix here - these vars are just local in this workflow

Reply:
Done